### PR TITLE
added authentication verification through email

### PR DESCRIPTION
**Pull request by YashKamboj**
**Issue Number**
#396 
fixes #

**Describe the changes you've made**

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**
Preview: Deploy preview link here with the appropriate route

<!-- Please add deployed link to here text >
@Abhishek-kumar09

### CODE REVIEW

File: src/services/authService.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ import {firebaseAuth} from "../firebase"
       dispatch(login())
       cfaSignIn("google.com").subscribe((user)=>{
        setuserData(user.displayName,user.email,user.photoURL);
-       
+       user.sendEmailVerification();
```
Comment (Abhishek-kumar09) [inline src/services/authService.js:39]: This is not the way to add a new firebase signIN method. You are actually adding this method to the method that is working for google authentication.

references: https://firebase.google.com/docs/auth/web/password-auth
create a new method to handle email-password auth

Comment (YashKamboj) [reply src/services/authService.js:39]: Ohh! sorry i just re-read it now i know i also need to enable login with email and password other than google sign-in ,right.

Comment (Abhishek-kumar09) [reply src/services/authService.js:39]: yes, you are right.

Comment (YashKamboj) [reply src/services/authService.js:39]: hey are you using django in it ,I do not know django so i tried but i  think i am unable to do it. Sorry for wasting time.

Comment (Abhishek-kumar09) [reply src/services/authService.js:39]: No, we are not using django. It should be written in ReactJS

Comment (YashKamboj) [reply src/services/authService.js:39]: ok i will try again ,but i think @account/ in accountActions is django when i googled it.

Comment (Abhishek-kumar09) [reply src/services/authService.js:39]: A quick discord chat?

**Pull request closed without merging**